src/types/env.ts

```diff
@@ -4,6 +4,7 @@ import { StaticDecode } from "@sinclair/typebox";
 export const envSchema = T.Object({
   SUPABASE_URL: T.String(),
   SUPABASE_KEY: T.String(),
+  APP_ID: T.String(),
```

Comment:
```suggestion
  APP_ID: T.Number(),
```

These are numbers why not encode them as such?

Reply:
You should comment on these to the related PR, I did not authored these.
https://github.com/ubiquibot/command-start-stop/pull/21

Reply:
I am confused because its not included in that pull. Besides, your pull indicates the change, or else it wouldn't appear on this diff.

Reply:
That's why I have it as a draft, and in the description I put "Depends on #21" which has to be merged and this one rebased. If it is not included then I forked from the wrong branch.